fenix/explores/fenix_feature_usage_events.explore.lkml

```diff
@@ -1,15 +1,16 @@
-include: "../views/fenix_feature_usage_events.view.lkml"
+include: "../views/feature_usage_events.view.lkml"
 include: "../views/fenix_dau.view.lkml"
 
 explore: fenix_feature_usage_events {
+  from: feature_usage_events
 
   sql_always_where: ${fenix_dau.submission_date} >= '2022-01-01'
       AND ${fenix_feature_usage_events.submission_date} >= '2022-01-01'
       ;;
 
   join: fenix_dau {
     type: inner
-    relationship: one_to_one
+    relationship: many_to_one
```

Comment:
This join will mess up aggregations on DAU, right? The join is on submission date, so when calculating a total, it'll be a {Fenix DAU on that date} * {# of combinations of dimension values}